projects/js-packages/publicize-components/src/social-store/constants.js

```diff
@@ -0,0 +1,7 @@
+export const CONNECTION_SERVICE_FACEBOOK = 'facebook';
+export const CONNECTION_SERVICE_INSTAGRAM_BUSINESS = 'instagram-business';
+export const CONNECTION_SERVICE_LINKEDIN = 'linkedin';
+export const CONNECTION_SERVICE_MASTODON = 'mastodon';
+export const CONNECTION_SERVICE_NEXTDOOR = 'nextdoor';
+export const CONNECTION_SERVICE_TUMBLR = 'tumblr';
+export const CONNECTION_SERVICE_TWITTER = 'twitter';
```

Comment:
Any particular reason for the separate file? It seems like they should coexist with the SOCIAL_STORE_ID.

Reply:
Done